app/modules/users/selectors.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import type { State } from 'types/state';
+import type { Identifier } from 'types/model';
+import type { User, UsersState, DisplayNameAndEmailType } from './model';
+
+const getModule = (state: State): UsersState => {
+  return state.modules.users;
+};
+
+export const getById = (state: State, id: Identifier): User => {
+  return getModule(state)[id];
+};
+
+// eslint-disable-next-line
```

Comment:
Please specify the specific rule(s) that you want to disable instead of disabling them all; that might allow errors to go undetected.